src/App/Entity/Talk.php

```diff
@@ -0,0 +1,113 @@
+<?php
+declare(strict_types=1);
+
+namespace App\Entity;
+
+class Talk
+{
+    /**
+     * @var string
+     */
+    private $speakerName;
+
+    /**
+     * @var string
+     */
+    private $speakerTwitter;
+
+    /**
+     * @var string
+     */
+    private $talkName;
+
+    /**
+     * @var string
+     */
+    private $abstract;
+
+    public function __construct(string $speakerName, string $speakerTwitter, string $talkName, string $abstract = null)
+    {
+        $this->speakerName = $speakerName;
+        $this->speakerTwitter = $speakerTwitter;
+        $this->talkName = $talkName;
+        $this->abstract = $abstract;
+    }
+
+    public function __toString() : string
```

Comment:
I don't really like having HTML here, but that's just me and not anything new in this PR ;)